types/reefjs/v7/index.d.ts

```diff
@@ -0,0 +1,173 @@
+// Type definitions for reefjs 7.6
```

Comment:
Keeping the v7 types around is only needed if you will need to publish changes to `@types/reefjs@7` in the future. I'd guess that's not needed for reefjs types, but it's up to you.